### ISSUE TITLE
Implement output component

### ISSUE THREAD

**Issue by anshumanv** [anshumanv/evolutions-webapp#15]
##### This issue is about adding an output/result component when evolve button is clicked.

- [x] I ensure that this issue is not a duplicate of any current open issues(s).
- [ ] Assign this issue to me (I want to work on this).
- [x] I will send my PR only after I'm assigned to this issue by any maintainer.
- [x] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.
